src/test/resources/allure.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name="Test Suite" verbose="2">
+    <listeners>
+        <listener class-name="com.coherentsolutions.training.aqa.java.web.nikonova.utils.FailedTestListener"/>
+    </listeners>
+    <test name="Allure Tests">
+        <classes>
+            <class name="com.coherentsolutions.training.aqa.java.web.nikonova.objects.AutoWishListTest"/>
+            <class name="com.coherentsolutions.training.aqa.java.web.nikonova.objects.CartTest"/>
+            <class name="com.coherentsolutions.training.aqa.java.web.nikonova.objects.CustomWishlistTest"/>
+            <class name="com.coherentsolutions.training.aqa.java.web.nikonova.objects.FailedTestsForScreenshots" />
+            <class name="com.coherentsolutions.training.aqa.java.web.nikonova.objects.UserLoginTest"/>
+            <class name="com.coherentsolutions.training.aqa.java.web.nikonova.objects.UserRegistrationFormPage"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
very weird filename for this suite(allure.xml) . You have some set of tests that do not test allure. Perhaps this is some kind of smoke tests or regression tests, so it’s better to name this file in accordance with the one you are testing here